Indicate that you get Files, not Responses (#32)
inrupt/docs-client-lib-js: source/tutorial/read-write-files.rst

Like anything else in your Pod, each file is a resource with a distinct
URL, which may or may not contain a hint such as a ``.jpg`` extension
for a photo. Files are represented as `Blobs
<https://developer.mozilla.org/docs/Web/API/Blob>`_.

Read a File
===========

> Reading a file is just a matter of fetching the content available at a
> certain URL. You get the fetched file as a blob. It is then up to you to decode
> it appropriately.

Reviewer: ```suggestion
Reading a file fetches the content at the specified URL as a `Blob <https://developer.mozilla.org/docs/Web/API/Blob>`_.  Once fetched, decode appropriately.
```
